app/experimenter/static/rapid/components/experiments/ExperimentDetails.tsx

```diff
@@ -68,31 +69,54 @@ const ExperimentDetails: React.FC = () => {
     );
   }
 
-  const launched_statuses = [ExperimentStatus.LIVE, ExperimentStatus.COMPLETE];
+  const buttonsDisabled = experimentData.status !== ExperimentStatus.DRAFT;
+  const buttonsShown = ![
+    ExperimentStatus.LIVE,
+    ExperimentStatus.COMPLETE,
+  ].includes(experimentData.status);
+
   let changeStatusButtons;
-  if (!launched_statuses.includes(experimentData.status)) {
-    changeStatusButtons = (
-      <div className="d-flex mt-4">
-        <span>
-          <Link
-            className="btn btn-secondary"
-            to={`/${experimentData.slug}/edit/`}
-          >
-            Back
-          </Link>
-        </span>
+  if (buttonsShown) {
+    if (buttonsDisabled) {
```

Comment:
could we reduce this is else for button disabled to:
```
      <div className="d-flex mt-4">
        <span>
          <Link to={`/${experimentData.slug}/edit/`}>
            <button className="btn btn-primary" disabled={buttonsDisabled}>
              Back
            </button>
          </Link>
        </span>

        <span className="flex-grow-1 text-right">
          <button
            disabled={buttonsDisabled}
            className="btn btn-primary"
            type="button"
            onClick={handleClickRequestApproval}
          >
            Request Approval
          </button>
        </span>
      </div>
```
?

The disable buttons would be blue though?

Reply:
We could use the [**classnames**](https://www.npmjs.com/package/classnames) npm module to conditionally set the `className` values to toggle between "btn-primary" and "btn-secondary".

Reply:
Lol you'd think!  That only works in `react-bootstrap`, the Link component we're using doesn't have it:

```
Type '{ children: string; disabled: boolean; className: string; to: string; }' is not assignable to type 'IntrinsicAttributes & LinkProps<PoorMansUnknown> & RefAttributes<HTMLAnchorElement>'.
  Property 'disabled' does not exist on type 'IntrinsicAttributes & LinkProps<PoorMansUnknown> & RefAttributes<HTMLAnchorElement>'.ts(2322)
```

but actually I just found another hack:

```js
            to={buttonsDisabled ? "#" : `/${experimentData.slug}/edit/`}
```

it still shows the hand cursor as if it's clickable, stackoverflow says to disable the pointer using custom CSS but meh.  We should just add react bootstrap that has the correct attributes.